### DISCUSSION TITLE
Tank level sensor

### DISCUSSION THREAD

**Discussion by yamirWolf** [Koenkk/zigbee2mqtt#24289] [Devices]
Hello everyone, I have a question/problem.
I have a question/problem. I have a Tuya level sensor called ME201WZ with zigbee connection. The system tells me the following:
“The device 'Oxa4c1389c1bf66286' is only compatible with revision '21' of the ZigBee specification (current revision: 23)”.

I am trying to follow the steps given at (https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#support-new-devices) but I can't get it to work.
Could you please help me?

Thanks!
![IMG_7569](https://github.com/user-attachments/assets/13ff6a77-4388-4ff0-8399-8c7acc4d13e4)
<img width="668" alt="Captura de pantalla 2024-10-11 a las 12 54 13" src="https://github.com/user-attachments/assets/270f3d21-8800-4daa-a6d8-30191b955713">
<img width="262" alt="Captura de pantalla 2024-10-11 a las 12 54 46" src="https://github.com/user-attachments/assets/a36bfe10-3751-4187-baa1-c17ef70798de">

**Comment by KreeQ76** [comment]
Same problem here..
